coralproject/talk
# Auto close comment toggle

## What does this PR do?

Default to not auto-closing a comment stream after 2 weeks.

## How do I test this PR?

- create a new asset
- check the db to make sure there's no `closedAt` date set on the record.
